app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -387,6 +417,7 @@ let state = {
         option: `no_with_veto`
       }
     ],
+    2: [],
```

Comment:
do the properties reflect the propsal_ids? should they then maybe be strings?

Reply:
good point, I'll try switching them to strings

Reply:
keys can't be strings. I get a `unexpected token ...` error

Reply:
Did you use backticks or quotemarks? Keys are not allowed to have backticks.

Reply:
done